source/dlangbot/cron.d

```diff
@@ -138,10 +138,11 @@ mergable:
 auto detectPRWithPersistentCIFailures(PRTuple t)
 {
     // label PR with persistent CI failures
-    auto status = t.pr.status;
-    auto failCount = status.filter!((e){
-        if (e.state == GHCiStatus.State.failure ||
-            e.state == GHCiStatus.State.error)
+    // TODO: unclear whether we want all statuses for PR commit, or only the latest
+    auto status = t.pr.combinedStatus;
+    auto failCount = status.latestStatuses.filter!((e){
```

Comment:
FWIW both are the same:

https://api.github.com/repos/dlang/phobos/commits/a04acd6a2813fb344d3e47369cf7fd64523ece44/status
https://api.github.com/repos/dlang/phobos/status/a04acd6a2813fb344d3e47369cf7fd64523ece44

Reply:
Yes, I just went with the one from the API docs.